examples/to-do/index.es

```diff
@@ -0,0 +1,78 @@
+Element `to-do`
+
+(class extends Storable (HTMLElement) {
+
+  initialize
+    (tasks = { tasks: [{ task: 'Eat Food' }] })
+      { this.retrieve (tasks) }
+
+
+  onsubmit (event, input = this.select `input`) {
+    event.preventDefault () // ...form from submitting
+
+    this.context.tasks.push ({ task: input.value })
+
+    this.store () // save context
+    input.value = ''
+  }
+
+
+  onidle (mark) {
```

Comment:
when does this run? frequently? seems like it might be doing wasteful work if so. might be able to optimize with a dirty checking mechanism for eager exit